src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  let discount = 0;
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    discount = SHORT_TERM_DISCOUNT;
+  } else if (days >= LONG_TERM) {
+    discount = LONG_TERM_DISCOUNT;
+  } else {
+    discount = 0;
+  }
+
+  const totalCost = days * 40 - discount;
```

Comment:
The daily cost '40' should be extracted to a constant to avoid hardcoding values. For example, 'const DAILY_COST = 40;' can be declared at the top with other constants.